### PR TITLE
feat: support customizing page icon

### PR DESCRIPTION
**Pull request by LucasXu0**
### Feature Preview

- sort emoji by group
- support changing emoji skin tone
- add more emojis
- customize the view icon and sync the icon between page and view
- redesign the emoji picker

https://github.com/AppFlowy-IO/AppFlowy/assets/11863087/606ca059-0143-4803-9bac-f919e2849452

Screenshot 2023-10-31 at 22 58 53

---

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -70,11 +69,6 @@ jobs:
           components: rustfmt
           profile: minimal
 
-      - uses: Swatinem/rust-cache@v2
-        with:
-          prefix-key: appflowy-lib-cache
-          key: ${{ matrix.job.os }}-${{ matrix.job.target }}
-
```
Comment (Xazin) [inline .github/workflows/release.yml:77]: Are these changes intentional?

Comment (LucasXu0) [reply .github/workflows/release.yml:77]: Yes, we shouldn't use cache when building the release package, as it may encounter errors.

**Comment by LucasXu0** [comment]
Redefine the emoji item builder.

Screenshot 2023-11-01 at 11 52 09

**Comment by LucasXu0** [comment]
Screenshot 2023-11-01 at 17 22 23
